contracts/market/src/contract.rs

```diff
@@ -119,6 +121,10 @@ pub fn execute(
                 fee_increase_factor,
             )
         }
+
+        ExecuteMsg::UpdateFeeFlashMint { fee_flash_mint } => {
+            update_fee_flash_mint(deps, info, fee_flash_mint)
```

Comment:
Move this into `update_config` and check that the fee is greater then zero and less then 1

Reply:
Fixed